File: INTERACTIVITYDEMO/app/src/main/java/com/example/mahtab/interactivitydemo/MainActivity.java
package com.example.mahtab.interactivitydemo;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.EditText;


public class MainActivity extends AppCompatActivity {

    public void clickfunction(View view){
        EditText nameEditText = (EditText)findViewById(R.id.nameEditText);

        Log.i("Information","You have succesfully clicked which means this button is working properly");
        Log.i("Values",nameEditText.getText().toString());

    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: ExampleApplication/app/src/main/java/com/example/exampleapplication/MainActivity.kt
package com.example.exampleapplication

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    public fun clickFunction(View view) {}
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}
